experiment.krun

```diff
@@ -20,36 +20,48 @@ sys.path.insert(0, DIR)
 # Executables
 YKSOM_BASELINE_BIN = os.path.join("yksom", "yksom_baseline", "release", "yksom")
 YKSOM_CUSTOM_RUSTC_BIN = os.path.join("yksom", "yksom_custom_rustc", "release", "yksom")
+SOMPP_BIN = os.path.join("SOMpp", "build", "SOM++")
 
 # Classpaths
 SOM_STD_LIB  = os.path.join("yksom", "SOM", "Smalltalk")
+KRUN_PATH  = os.path.join("benchmarks")
 SOM_BENCHMARK_DIR  = os.path.join("yksom", "SOM", "Examples", "Benchmarks")
 SOM_BENCHMARK_SUBDIRS = [os.path.join(SOM_BENCHMARK_DIR, d) for d in os.listdir(SOM_BENCHMARK_DIR) \
                             if os.path.isdir(os.path.join(SOM_BENCHMARK_DIR, d))]
-SOM_CLASSPATHS = [SOM_STD_LIB, SOM_BENCHMARK_DIR] +  SOM_BENCHMARK_SUBDIRS
+SOM_CLASSPATHS = [SOM_STD_LIB, SOM_BENCHMARK_DIR] +  SOM_BENCHMARK_SUBDIRS + KRUN_PATH
 
 N_EXECUTIONS = 30
 ITERATIONS_ALL_VMS = 2000
 
+LIBKRUN = os.path.join("krun", "libkrun")
+
+N_EXECUTIONS = 10
+ITERATIONS_ALL_VMS = 200
```

Comment:
These two declarations seem to override the declarations on lines 33/34?

Reply:
Doh! Good catch. This was debugging code that I checked in by mistake.

Reply:
Fixed: a73c856e136094e66a9672cf1c3eb196748980cd